nncf/config/schemata/experimental_schema.py

```diff
@@ -13,17 +13,17 @@
 
```

Comment:
@ljaljushkin I did not touch this for the time being, but this will have to be ultimately rechecked and defaults set up in the same manner as in other parts of the config. I suppose that this will be done once NAS moves out of experimental

Reply:
Is experimental schema verified on config loading now?

Reply:
I didn't check explicitly, but judging from the code - yes, as it was before.